main.go

```diff
@@ -46,5 +53,14 @@ func main() {
 	// Register template functions.
 	beego.AddFuncMap("i18n", i18n.Tr)
 
+	// serve securely if the certificates are present
+	_, certErr := os.Stat(servingCertFile)
+	_, keyErr := os.Stat(servingKeyFile)
+	if certErr == nil && keyErr == nil && len(servingCertFile) > 0 && len(servingKeyFile) > 0 {
```

Comment:
Truth be told, I'd rather fail hard if they specified either env var and something (anything) wasn't right with either envvar.  That would help guide users toward setting it properly as opposed to silently ignoring it.

It's not my baby though.  If you'd like that check method I'll do it.  Let me know.